### PR TITLE
Feature/jkw focus modal

### LINKED ISSUE #86: Break out 'Time to focus' modal into own component
### Feature Categories:
- React
- Styling/CSS

**Describe the feature/Bug your gonna work on**
- Now that we've incorporated mood rating functionality into our app, we need to move our `Time to focus` message to after the second mood rating.
   - This includes: testing, implementation, and styling

**Additional context**
- We may also move our `postSession` request into this component as well, as a way to "tie up" the end of the session! This may DRY up some of the if conditions in `MoodRating` (tbd)
- (Maybe) we include a session history in this component so the user sees the cycle they just completed.

### PR DESCRIPTION
**Pull request by jkwest-93**
# Som Timer Pull Request Form
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- After adding mood functionality, we needed to change the placement of our "Time to focus again!" message. To do this, I created a `FocusModal` component, which maintains the styling from the prior implementation in our codebase. This component has a `useEffect` hook which sets it's lifespan (2 seconds) and posts our `session` data to the database.
     - This PR includes testing, implementation, and styling for the `FocusModal` component.
     - I refactored the `MoodRating` component to DRY up some conditionals and submit our session data as the timer transitions from `MoodRating` 2 back to `CountdownTimer`
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
- Ran `npm test` in the browser; see attached screenshot for demo of functionality.

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
userCycle_focusModal
- Closes #86

### CODE REVIEW

File: src/Components/FocusModal/FocusModal.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  })
+
+  return (
+    <div className={style.workMsgModal}>
+      <h2 className={style.modalContent}>
+        Time to focus again!
+      </h2>
+    </div>)
+}
+
```
Comment (Abdeboskey) [inline src/Components/FocusModal/FocusModal.js:23]: Love this cute lil component!

**Review by Abdeboskey — APPROVED** [review]
This looks great, it was a good idea to move this modal into its own component!!

File: src/Components/MoodRating/MoodRating.js [hunk 2]
```diff
@@ -45,105 +42,108 @@ const MoodRating = () => {
...
-            onClick={rateMood}
-            name={1}
-            aria-label="Rate mood 1 out of 5"
-          >
-            <img 
-              className={style.moodIcon}
-              src={moodIcon1} 
-              alt="Unhappy face"
+    <>
+      <section className={!sessionComplete ? style.moodRatingContainer : style.moodRatingModal}>
```
Comment (rwilliams659) [inline src/Components/MoodRating/MoodRating.js:51]: Great job with the modal rendering here!

File: src/Components/MoodRating/MoodRating.module.scss [hunk 3]
```diff
@@ -39,6 +39,11 @@
   align-items: center;
 }
 
+.moodRatingModal {
+  @extend .moodRatingContainer;
+  -webkit-filter: blur(5px) grayscale(50%);
```
Comment (rwilliams659) [inline src/Components/MoodRating/MoodRating.module.scss:44]: Love the background blur! Looks awesome

File: src/Components/FocusModal/FocusModal.test.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  it('Should make a network request when rendered', () => {
+    render(
+      <ViewProvider>
+        <SessionProvider>
+          <FocusModal />
+        </SessionProvider>
+      </ViewProvider>
+    )
+
+    expect(postSession).toBeCalled()
```
Comment (rwilliams659) [inline src/Components/FocusModal/FocusModal.test.js:33]: Glad we were able to test postSession in the unit test!

**Review by rwilliams659 — APPROVED** [review]
Love that we got the modal re-implemented after mood2!

**Pull request merged**